plugin/clone/src/clone_client.cc

```diff
@@ -1056,10 +1059,18 @@ int Client::validate_remote_params() {
     auto &plugin_name = plugin.first;
     auto &so_name = plugin.second;
 
+    if (plugin_name.compare(ROCKSDB) == 0) donar_rocksdb_installed = true;
+
     if (plugin_is_installed(plugin_name)) {
       continue;
     }
 
+    /* Donor installs ROCKSDB and client doesn't. Fail the clone. */
```

Comment:
The commit message / PR description needs updating, as it might be misleading: the code checks for plugins, the commit message talks about default storage engine, which is an entirely different thing (a server sysvar).